src/main/java/org/scijava/log/StderrLogService.java

```diff
@@ -49,18 +51,12 @@
 @Plugin(type = Service.class, priority = Priority.LOW_PRIORITY)
 public class StderrLogService extends AbstractLogService {
 
+	private final LogFormatter formatter = new DefaultLogFormatter();
+
 	@Override
 	public void alwaysLog(final LogLevel level, final Object msg, final Throwable t) {
-		final String message = level.prefix() + msg;
-		// NB: Emit severe messages to stderr, and less severe ones to stdout.
```

Comment:
I appreciate refactoring to make things easier to read and understand, but it is very important to keep the `NB` notes instead of throwing them away. Every `NB` note indicates something of particular note, and often unintuitive, to developers reading the code. See http://imagej.net/Coding_style#Javadoc_and_comments